src/components/network/line-layer.js

```diff
@@ -171,6 +172,23 @@ class LineLayer extends CompositeLayer {
         };
     }
 
+    getVoltageLevelIndex(voltageLevelId) {
+        const { network } = this.props;
+        const vl = network.getVoltageLevel(voltageLevelId);
+        const substation = network.getSubstation(vl.substationId);
+        return (
+            [
+                ...new Set(
+                    substation.voltageLevels.map((vl) => vl.nominalVoltage) // only one voltage level
+                ),
+            ]
+                .sort((a, b) => {
+                    return a - b; // force numerical sort
+                })
+                .indexOf(vl.nominalVoltage) + 0.1
```

Comment:
Add a comment to say that this means 10% of the radius. and why not 50% to be in the middle of the width of the ring of the voltage level ?

Reply:
Do we really need the +x% ? I don't see any problem without it.
If we don't have it, then it means that for substations with 1 voltage level we don't have to translate the source at all